Mathlib/Topology/Algebra/Order/LiminfLimsup.lean

```diff
@@ -554,3 +555,79 @@ theorem limsup_eq_tendsto_sum_indicator_atTop (R : Type _) [StrictOrderedSemirin
 #align limsup_eq_tendsto_sum_indicator_at_top limsup_eq_tendsto_sum_indicator_atTop
 
 end Indicator
+
+section LiminfLimsupAddSub
+
+variable {R : Type _} [ConditionallyCompleteLinearOrder R] [TopologicalSpace R] [OrderTopology R]
```

Comment:
```suggestion
variable {R : Type*} [ConditionallyCompleteLinearOrder R] [TopologicalSpace R] [OrderTopology R]
```
Sorry, the change to `Type*` is more recent than this PR so I missed it. Altough the current `bors` failure does not seem to be related, this would probably cause an error in the next batch...